otcextensions/tests/functional/sdk/dds/v3/test_instance.py

```diff
@@ -0,0 +1,162 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+import uuid
+
+from otcextensions.tests.functional.sdk.dds import TestDds
+
+
+class TestInstance(TestDds):
+    uuid_v4 = uuid.uuid4().hex[:8]
+    network = None
+    instance = None
+
+    def setUp(self):
+        super(TestInstance, self).setUp()
+
+    def create_network(self):
```

Comment:
can you please in the next patch make a general test function to create/delete nets with some params? We have it really now in lots of places

Reply:
ok, #208